util/dvsim/Scheduler.py

```diff
@@ -188,17 +182,12 @@ def _check_if_done(self, timer, hms, print_status):
 
     def run(self, timer, old_results):
         '''Run the jobs for this target.
-
         timer is a Timer that was started at the start of the Runner's run.
-
         old_results is a dictionary mapping items (from previous targets) to
         statuses. Every job that appears as a dependency will be in this list
         (because it ran as part of a previous target).
-
         is_first_tgt is true if this is the first target to run.
-
         Returns the results from this target (in the same format).
-
         '''
```

Comment:
As above, this change changes the formatting from being right to being wrong. Please don't.